### ISSUE TITLE
Page contents are mixing

### ISSUE THREAD

**Issue by bsanjaasuren** [boazsegev/combine_pdf#152]
Hi, I have two PDF pages generated by using combine pdf. And showing user_id, user_name each page. But in a very rare occurrence, contents are mixed in output. The version I am using is 0.2.5, I cannot figure out this issue because it is very rare. please help

**Rails code:**
```
pdf = CombinePDF.new
pdf << CombinePDF.new(pdf_page_1)
pdf << CombinePDF.new(pdf_page_2)
send_data(pdf.to_pdf...)
```

**PDF outputs:** 

```
correct information is: 
page1: #1 user1
page2: #2 user2

wrong information:
page1: #2 user1
page2: #1 user2
```

My question is:
1. Is it possible on combine_pdf gem?
2. Do I have to update combine_pdf version?

**Comment by bsanjaasuren** [comment]
Thank for your quick response @boazsegev. I will update my gem 👍☺️  .

**Comment by bsanjaasuren** [comment]
Sorry I have edited my issue. And I guess it is very similar to issue #109.
We are both using prawn pdf, so I hope it is fixed by 1.0.1 verison of CombinePDF.
